packages/hoppscotch-app/components/app/PaneLayout.vue

```diff
@@ -43,7 +50,9 @@ import { Splitpanes, Pane } from "splitpanes"
 import "splitpanes/dist/splitpanes.css"
 import { breakpointsTailwind, useBreakpoints } from "@vueuse/core"
 import { computed, useSlots } from "@nuxtjs/composition-api"
+import { ref } from "@vue/runtime-dom"
```

Comment:
We need to import `ref` from `@nuxtjs/composition-api` instead of `@vue/runtime-dom`